EligibilityCheckActivity/handler.ts

```diff
@@ -73,6 +76,24 @@ export const getEligibilityCheckActivityHandler = (
           })
           .map(_ => ({ dsu: _, fiscalCode }));
       })
+      .chain(data => {
+        if (
+          !data.dsu.DatiIndicatore?.Componenti?.some(
+            familyMember => familyMember.CodiceFiscale === data.fiscalCode
+          )
+        ) {
+          return taskEither.of({
+            dsu: {
+              ...data.dsu,
+              DescrizioneError:
+                "Missing requester fiscal code inside family members array",
+              Esito: EsitoEnum.ERRORE_INTERNO
```

Comment:
if I remember well we trigger a retry in case of ERRORE_INTERNO but this shoulnd't be the case, can you please check this?